src/mame/drivers/galaxian.cpp

```diff
@@ -6864,17 +6926,33 @@ void galaxian_state::init_victoryc()
 
 void galaxian_state::init_victorycb()
 {
-	init_galaxian();
-
 	address_space &space = m_maincpu->space(AS_PROGRAM);
 
-	/* needs a full 2k of RAM */
+	/* same as galaxian... */
+	init_galaxian();
+
+	/* ..but needs a full 2k of RAM */
```

Comment:
This kind of change is undesirable - it's better to minimise the scope of variables and keep the delcaration/initialisation close to the point of use.  Also, the "same as galaxian" comment is pointless - it tells you no more than the line it's attached to.  Comments like this just add clutter.

Same applies to the other init functions.

The other issue is that this is a legacy code pattern that we're trying to discourage.  Things should be converted to use derived address maps, not add more code to adjust mapping in init methods.